### DISCUSSION TITLE
[Config Support]: Notification settings per camera

### DISCUSSION THREAD

**Discussion by SlavikCA** [blakeblackshear/frigate#14697] [Config Support] [answered]
### Describe the problem you are having

One of my camera looking to the street and I'm getting tons of notifications, which are not important. I want to disable notifications for this camera.

Second camera looking is doorbell. And I want notifications from that camera only.

Can I get notifications configured per camera?

### Version

frigate:0.15.0-beta1-tensorrt

**Comment by NickM-27** [comment, marked as answer]
you should use the config to make it so only things you deem important are considered alerts. notifications are only sent for alerts
